panic: runtime error: integer divide by zero

**Describe the bug**
panic: runtime error: integer divide by zero

**To Reproduce**
Steps to reproduce the behavior:
1. Uploaded already several hundred files with version 0.5
2. Upgraded to 0.8
3. See error

**Expected behavior**
Continue uploading files which are not yet marked as done (ok), but then panics.


**Desktop (please complete the following information):**
 - OS: Linux

```
2019/09/13 10:11:15 Getting new upload URL for '/mnt/backup/smartphone/to_backup/sdcard/0000-0000/DCIM/Camera/IMG_20180906_220318.jpg'
2019/09/13 10:11:16 Uploading content to 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2UrHh4inO2Pqt5RZk0iOruPhcHsSpY37ju--f_NbwPtu4FniZ6oZCxsAqdT4v-oRVAqNIob_8WbpqdPNWE_fYeckKLz8pRkSJxp8kpNtuDm18YTtEaA&upload_protocol=resumable'
[12%] Sent 212992 of 1723678 bytes (total file size: 1723678)2019/09/13 10:11:16 Uploading content to 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2Uqp0xeACOqEURtDNDzetce1TXfWAX_EKyrRcmDCFMcTYbmBW1VvKSz-M6XUy4C0hNAQQ9welSCoTGf7Qol9sxvXXeZJ3w&upload_protocol=resumable'
[7%] Sent 114688 of 1502884 bytes (total file size: 1502884)2019/09/13 10:11:16 Uploading content to 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2UqxqJG2GBuzF0fsovLJe3QSchs4-I1Jea46lwbwb2qnvXWXekU6Sn_t_5WsUsaVg0svA4rMsFmX0K3rdh_qWdLpD6CUWxdlTYgprLVIRqoSJVFrDbI&upload_protocol=resumable'
[3%] Sent 49152 of 1454153 bytes (total file size: 1454153)2019/09/13 10:11:16 Uploading content to 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2UrKdmmzaMnlMZBJfSyf4yT8KyTZnvzc53RAX-n_pWaWGrWxReR2ZHD71jJShP-V3TyI_Z70-nyk1NONCBe8siCOHWLOBuDjVhYAe-Gm32ELqrzYrp0&upload_protocol=resumable'
2019/09/13 10:11:16 Uploading content to 'https://photoslibrary.googleapis.com/v1/uploads?upload_id=AEnB2UoLqGOqt8o4H1nouVdLZAgwDBMNbrwsNVpKZRVtCB4fj3qGDcjXyS05I3ZWyOszXeXLlQqfqRy1BOvfF1zq2DlNudBL3EWFr2jAt1oHITUzmP8GW8E&upload_protocol=resumable'
[1%] Sent 16384 of 1568824 bytes (total file size: 1568824)panic: runtime error: integer divide by zero

goroutine 59 [running]:
github.com/gphotosuploader/google-photos-api-client-go/lib-gphotos.(*ReadProgressReporter).report(0xc00054d800)
	/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go@v1.0.3/lib-gphotos/progress.go:27 +0x212
github.com/gphotosuploader/google-photos-api-client-go/lib-gphotos.(*ReadProgressReporter).Read(0xc00054d800, 0xc001326000, 0x4000, 0x4000, 0xc0001b2e80, 0x20, 0xc0001b2e78)
	/go/pkg/mod/github.com/gphotosuploader/google-photos-api-client-go@v1.0.3/lib-gphotos/progress.go:22 +0x8e
net/http.(*http2clientStream).writeRequestBody(0xc000b0db80, 0x7f42282bdd40, 0xc000b11820, 0x7f42282bdd60, 0xc000b11820, 0x0, 0x0)
	/usr/local/go/src/net/http/h2_bundle.go:7694 +0x521
net/http.(*http2Transport).getBodyWriterState.func1()
	/usr/local/go/src/net/http/h2_bundle.go:8908 +0xc2
created by net/http.http2bodyWriterState.scheduleBodyWrite
	/usr/local/go/src/net/http/h2_bundle.go:8955 +0xa3
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/) and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## 0.8.2
+### Fixed
+- Fix panic when a very big file was uploaded. It was solved in upstream [gphotosuploader/google-photos-api-client-go#17](https://github.com/gphotosuploader/google-photos-api-client-go/issues/17). (#127)
+
 ## 0.8.1
 ### Added
 - Coverage reports in [codecov](https://codecov.io/gh/gphotosuploader/gphotos-uploader-cli/) service.

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.12
 require (
 	github.com/99designs/keyring v0.0.0-20190704105226-2c916c935b9f
 	github.com/client9/xson v0.0.0-20180321172152-0e50cdfc08c0
-	github.com/gphotosuploader/google-photos-api-client-go v1.0.4
+	github.com/gphotosuploader/google-photos-api-client-go v1.0.5
 	github.com/gphotosuploader/googlemirror v0.3.2
 	github.com/int128/oauth2cli v1.4.1
 	github.com/juju/errors v0.0.0-20190207033735-e65537c515d7

